ktaglib/.cxx/cmake/debug/arm64-v8a/android_gradle_build_targets.command.txt

```diff
@@ -0,0 +1,8 @@
+Executable : /home/somal/Android/Sdk/cmake/3.10.2.4988404/bin/ninja
+arguments : 
+-C
+/home/somal/GSoCProgress/MuBandroidnewtry/musicbrainz-android/ktaglib/.cxx/cmake/debug/arm64-v8a
```

Comment:
Maybe this `txt` with the path to your repo is not needed to be committed?

Reply:
no they are not required.
I will make the changes